ingest/ingest_pipeline.py

```diff
@@ -116,10 +120,14 @@ def __init__(
         self.study_file_id = study_file_id
         self.matrix_file = matrix_file
         self.matrix_file_type = matrix_file_type
+        self.extra_log_params = {'study_id': self.study_id, 'duration': None}
         if os.environ.get('DATABASE_HOST') is not None:
             # Needed to run tests in CircleCI.  TODO: add mock, remove this
+            self.error_logger.error(f"Database host is: {os.environ.get('DATABASE_HOST')}", extra=self.extra_log_params)
+            print(f"Database host is: {os.environ.get('DATABASE_HOST')}")
             self.db = self.get_mongo_db()
         else:
+            self.error_logger.error(f"Database host is: NONE", extra=self.extra_log_params)
```

Comment:
Having not-errors logged to an error log is confusing and seems likely to complicate debugging.

How about simply logging "info" and "errors" to the same file (perhaps "`run.log`" or some such)?  The SCP Rails app does this to good effect, and I think Cromwell does too.

Reply:
concur